File: backend/app/utils/format.py
def format_bytes(n: int) -> str:
    """将字节数格式化为可读的 KB/MB/GB/TB。"""
    if n < 1024:
        return f"{n} B"
    if n < 1024 ** 2:
        return f"{n / 1024:.1f} KB"
    if n < 1024 ** 3:
        return f"{n / (1024 ** 2):.1f} MB"
    if n < 1024 ** 4:
        return f"{n / (1024 ** 3):.2f} GB"
    return f"{n / (1024 ** 4):.2f} TB"

File: frontend/src/utils/format.ts
/**
 * 将字节数格式化为人类可读的字符串
 */
export function formatBytes(bytes: number): string {
  if (bytes < 1024) return bytes + ' B';
  if (bytes < 1024 * 1024) return (bytes / 1024).toFixed(1) + ' KB';
  if (bytes < 1024 * 1024 * 1024) return (bytes / (1024 * 1024)).toFixed(1) + ' MB';
  return (bytes / (1024 * 1024 * 1024)).toFixed(2) + ' GB';
}

/**
 * 将每秒字节数格式化为速度字符串
 */
export function formatSpeed(bytesPerSec: number): string {
  return formatBytes(bytesPerSec) + '/s';
}
